user_docs/en/changes.t2t

```diff
@@ -127,6 +129,14 @@ That method receives a ``DriverRegistrar`` object on which the ``addUsbDevices``
 - ``languageHandler.makeNpgettext`` and ``languageHandler.makePgettext`` have been removed.
 ``npgettext`` and ``pgettext`` are supported natively now. (#15546)
 - The app module for [Poedit https://poedit.net] has been changed significantly. The ``fetchObject`` function has been removed. (#15313, #7303, @LeonarddeR)
+- The following redundant types and constants have been removed from ``hwPortUtils``:  (#15764, @LeonarddeR)

```

Comment:
just confirming, have you searched NVDA for usage of these? there doesn't appear to be any wider changes in this diff.

Reply:
Yes I have. These constants were probably copied from the scanner in the pyserial module.